src/cljs/vanilla/events.cljs

```diff
@@ -54,15 +55,17 @@
                          :data-grid {:x 0 :y 0 :w 5 :h 15})]
 
       (do
-        (prn ":add-widget " new-widget-type
-          " //// widget-type " widget-type
-          " //// named-widget " named-widget)
+        ;(prn ":add-widget " new-widget-type
+        ;  " //// widget-type " widget-type
+        ;  " //// named-widget " named-widget)
         (assoc db
           :widgets (conj (:widgets db) named-widget)
           :next-id (inc next-id))))))
 
 
 
+
+
 (rf/reg-event-db
```

Comment:
this event should be moved to core.cljs (maybe we should move all this handling to a new namespace?), closer to where it is used

Reply:
Sorry github is showing this wonky, which event(s)?  Remove-widget and update-layout, and add-widget?

Reply:
Got your email, will refactor where these events live accordingly